documentation/modules/ROOT/pages/integrations/outbox.adoc

```diff
@@ -239,6 +239,12 @@ for example, `com.company.TheAttributeConverter`
 |string
 |
 
+|[[quarkus-debezium-outbox-payload-explicit-type]]<<quarkus-debezium-outbox-payload-explicit-type,`+quarkus.debezium-outbox.payload.explicit-type+`>>::
+A fully-qualified class name of the explicit type of the payload. +
```

Comment:
I'm stumbling upon the term "explicit type" (in the docs, but also in the code). How about just "type"? Or "custom type"? Here in the docs, there also should be more clarification what this refers to exactly, e.g. saying "a fully-qualified class name of a Hibernate ORM user type implementation", linking to https://docs.jboss.org/hibernate/orm/current/userguide/html_single/Hibernate_User_Guide.html#basic-custom-type.

Reply:
This mostly comes from working with Hibernate where type often refers to the resolved type done by ORM and explicit being the one supplied by the user.  I'll adjust the naming here to be less explicit (pardon the pun) :P